### DISCUSSION TITLE
게시글 댓글 작성 폼에서 태그 표현방법 같이 고민해주세요 !!

### DISCUSSION THREAD

**Discussion by lion0913** [likelion-backendschool/your-anmok#51] [Q&A] [answered]
문제상황 : 번개게시글 상세 - 댓글을 태그해서 답글을 달때 표현방식이 고민입니다!

1. 뱃지를 사용한 닉네임 태그
<img width="587" alt="image" src="https://user-images.githubusercontent.com/63444424/187017659-98a06bd2-df8f-4a5c-943a-43ae9a70a0e4.png">
만약에 이 방식으로 갈 거라면 폼 안에 뱃지를 넣어서 인스타처럼..? 표현할 생각입니다

<br/>
2. 답댓글 작성중 ... 표현하기
<img width="573" alt="image" src="https://user-images.githubusercontent.com/63444424/187017624-ca24c8ad-2728-4fe9-8910-e7a0f634d132.png">

**Comment by joong2043** [comment, marked as answer]
첫번째꺼가 더 이쁘긴 합니다만 인풋 폼 안에 뱃지가 들어가는게 더 좋을 거 같습니다!

**Comment by ap3334** [comment]
뱃지가 더 예쁠거 같습니당 !
